presto-main/src/test/java/io/prestosql/sql/planner/TestPredicatePushdown.java

```diff
@@ -57,6 +57,17 @@
         super(ImmutableMap.of(ENABLE_DYNAMIC_FILTERING, "true"));
     }
 
+    @Test
+    public void testPushdownBindToNull() {
```

Comment:
This failure is unrelated to predicate pushdown. It's due to the `null` being inlined in the argument to the internal `bind` function that's used to model `try` when the `InlineProjections` optimizer rule executes. The query can be simplified to the following and still reproduce the issue:

```
SELECT try(k)
FROM (SELECT null) t(k)
```

Let's add a test using the simplified query in `io.prestosql.sql.query.TestExpressions` instead. Also add a comment in the test linking to the github issue so that we can trace back the reason for the test in the future.